Designs/UI/SearchLocation.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>WeConnect</title>
+</head>
+<body>
+    <div class="container">
+            <div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom box-shadow">
+                    <h5 class="my-0 mr-md-auto font-weight-normal">WeConnect</h5>
+                    <nav class="my-2 my-md-0 mr-md-3">
+                      <a class="p-2 text-dark" href="#">Dashboard</a>
+                      <a class="p-2 text-dark" href="#">Business Registration</a>
+                      <a class="p-2 text-dark" href="#">Business Reviews</a>
+                      <a class="p-2 text-dark" href="#">User Profile</a>
+                      <a class="p-2 text-dark" href="#">Business Catalog</a>
+                      <a class="p-2 text-dark" href="#">Log Out</a>
+
+                    
+                    </nav>
+                  </div>
+        </div>
+
+        <div class="container">
+                <form class="form-inline mt-2 mt-md-0 ">
+                        <input class="form-control mr-sm-2 search" type="text" placeholder="Search" aria-label="Search">
+                        <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search by Location</button>
+                      </form>
+
+                    <div>
+                        <img src="img/googleMaps.jpg" alt="Google Maps" >
+                    </div>
+        </div>
+
+        <div id="footer">
+           <p> copywright &copy; 2018: Author Ian Mwangi</p>
+        </div>
+    </div>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+</body>
+</html>
```

Comment:
Research on how to indent your HTML code, It should look much cleaner

Reply:
working on it